src/popover/index.js

```diff
@@ -78,7 +78,7 @@ class Popover extends Component {
   }
 
   _getBackDropElement = () => {
-    if (!this.props.onClickOut) return null;
+    if (!this.props.onClickOut || !this.props.isOpen) return null;
```

Comment:
won't this add it regardless if there is a click out function passed or not?

shouldn't it be something like this:

!this.props.onClickOut || this.props.onClickOut && !this.props.isOpen